extensions/hp-extension/src/hpHeart.ts

```diff
@@ -0,0 +1,245 @@
+export default {
+  id: 'LV Function Heart',
```

Comment:
This is the example hanging protocol that lays out an MR heart study in 2x2 mode with the following layout:
LAX 4 chamber      LAX 3 chamber
LAX 2 chamber      SAX
All views are "cine" views, that is containing multiple images, but also marked as CINE in the series description.